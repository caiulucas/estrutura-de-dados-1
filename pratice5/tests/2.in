5
Jones
Pedro
Carlos
Lucas
Juca
