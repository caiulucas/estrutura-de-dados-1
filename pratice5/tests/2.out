Juca
Lucas
Carlos
Pedro
Jones
